Makefile

```
@@ -53,6 +53,14 @@ TIMESTAMP:=$${__TIMESTAMP:-$(shell /bin/date -u "+%Y%m%d%H%MZ")}
 SHORT_SHA=`git rev-parse --short HEAD`
 BRANCH=`git rev-parse --abbrev-ref HEAD | tr -- '-/' '_'`
 
+# The dependent framework PR connection will be taken from the top commit's depends-on message.
+REQ_LEAPP_PR=$(shell git log -n 1 HEAD | grep '^.*Depends-On:.*[[:digit:]]*$$' | grep -Eo '*[[:digit:]]*')
```

That could hit even string like " 232  Depends-on: 333 PR. but takes 555... 555" - where the result will be multiple numbers:
```bash
echo  " 232  Depends-On: 333 PR. but takes 555... 555"  | grep '^.*Depends-On:.*[[:digit:]]*$' | grep -Eo '*[[:digit:]]*'
232
333
555
555
```

```suggestion
REQ_LEAPP_PR=$(shell git log -n 1 HEAD | grep -E '^[[:space:]]*Depends-On:[[:space:]]*[[:digit:]]+[[:space:]]*$$' | grep -Eo '*[[:digit:]]*')
```

---

@pirat89 this won't match the `Depends-On: https://direct/link/to/pr/XXX`.
Although I like the idea of making regex stricter, thanks, I guess `grep -m1 -iE '^[[:space:]]*Depends-On:[[:space:]]*.*[[:digit:]]+[[:space:]]*$$'` should be ok for both cases (url and just the number)

---

Should be fixed now